app/models/assignment.rb

```diff
@@ -26,7 +27,21 @@ class Assignment < ApplicationRecord
   scope :created_before, ->(max_time) { where('assignments.created_at < ?', max_time) }
   scope :created_after, ->(min_time) { where('assignments.created_at > ?', min_time) }
 
-  scope :ended, (-> { where('assignments.period_end < ?', Time.zone.today) })
+  scope :active, lambda {
```

Comment:
Currently we have those concerns defined in a concern. If they are to be moved in the models again, then they should be deleted from the concern.

Reply:
they have `assignments.created_at` in where, so will only work on Assignment, and no where else